openapi-cli/src/test/resources/generators/client/ballerina/request_body_with_ref.bal

```diff
@@ -38,7 +38,7 @@ public isolated client class Client {
     #
     # + payload - Return from creating a pet
     # + return - Successful operation
-    remote isolated function createPet(CreatedPet payload) returns http:Response|error {
+    remote isolated function createPet(CreatedPet_requestBody payload) returns http:Response|error {
```

Comment:
Using title case for the suffix looks better to me. WDYT? @lnash94 
```suggestion
    remote isolated function createPet(CreatedPet_RequestBody payload) returns http:Response|error {
```

Reply:
+1, I will update with the title case in the suffix